duo_wordpress.php

```diff
@@ -156,6 +156,11 @@ function duo_auth_enabled(){
             return false; //allows the XML-RPC protocol for remote publishing
         }
 
+        if (defined('REST_REQUEST') && REST_REQUEST && duo_get_option('duo_rest', '') == 'off') {
```

Comment:
It looks like we can probably use this filter: https://github.com/WordPress/WordPress/blob/27aa0664df927f610e42f9a694eb452cee49e245/wp-includes/rest-api/class-wp-rest-server.php#L267

Or after 4.7: https://github.com/WordPress/WordPress/blob/27aa0664df927f610e42f9a694eb452cee49e245/wp-includes/rest-api/class-wp-rest-server.php#L140